### PR TITLE
Exams by Maksym Rybalko, IM-22 group

### PR DESCRIPTION
**Pull request by M-Rybalko**

### CODE REVIEW

**Comment by KobbAsa** [comment]
After the review, I can say I agree with all the commits and no comment.

**Comment by radichenko** [comment]
I have reviewed the changes and they look good to me. Approving this pull request

File: .eslintrc.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     ],
     "linebreak-style": [
       "error",
-      "unix"
+      "windows"
```
Comment (tshemsedinov) [inline .eslintrc.json:21]: https://youtu.be/W87wOCSPA08

File: Tasks/filter.js [hunk 2]
```diff
@@ -1,15 +1,13 @@
...
-  return T;
+// Step 5
+// Use Array.prototype.filter()
+
+'use strict';
+
+const filter = (array, type) => {
+  const filtered = array.filter((element) => typeof element === type);
+  return filtered;
 };
```
Comment (tshemsedinov) [inline Tasks/filter.js:11]:
```suggestion
const filter = (array, type) => array
  .filter((element) => typeof element === type);
```

File: Tasks/sum.js [hunk 3]
```diff
@@ -1,20 +1,17 @@
...
-  return sum[sum.length - 1];
-  sum.push(sum.length - 1);
-});
+// Step 6(1)
+// Implement with Array.map()
+
+'use strict';
+
+const sum = (array) => {
+  let sum = 0;
```
Comment (tshemsedinov) [inline Tasks/sum.js:9]: Same name for function and variable will work but it's not of for human readability

File: Tasks/sum.js [hunk 4]
```diff
@@ -1,20 +1,17 @@
...
+  const condition = (element) => {
+    if (typeof element === 'number') sum += element;
+  };
+  array.map(condition);
```
Comment (tshemsedinov) [inline Tasks/sum.js:13]: Use `map` only to create new array as a result not for iteration, it's better to use for..of instead

File: Tasks/filter.js (hunk 2 above)
Comment (M-Rybalko) [reply Tasks/filter.js:11]: Reworked that 
https://github.com/M-Rybalko/Exams/tree/exam

File: Tasks/sum.js (hunk 3 above)
Comment (M-Rybalko) [reply Tasks/sum.js:9]: Reworked that 
https://github.com/M-Rybalko/Exams/tree/exam

**Pull request closed without merging**